src/codegen/query.cpp

```diff
@@ -11,11 +11,13 @@
 //===----------------------------------------------------------------------===//
 
 #include "codegen/query.h"
+#include "codegen/query_result_consumer.h"
 
 #include "codegen/query_parameters.h"
 #include "common/logger.h"
 #include "common/timer.h"
 #include "executor/executor_context.h"
+#include "executor/plan_executor.h"
```

Comment:
Try to follow style guide for header inclusion ordering ([here](https://google.github.io/styleguide/cppguide.html#Names_and_Order_of_Includes))